src/components/HeaderBar.js

```diff
@@ -65,17 +65,17 @@ class HeaderBar extends React.Component {
             <div>
                 <NavLink activeStyle={{ textShadow: "2px 2px 5px #5DADE2" }} exact to="/">
                     <li>
-                        <i className={ ` fas fa-home ${ this.props.darkMode ? "dark-mode-icon" : "" } ` }>&nbsp;Home</i>
+                        <i className={ ` fas fa-home ${ this.props.darkMode ? "dark-mode-icon" : "" } ` }><span className="fa-icon-inner-text">&nbsp;Home</span></i>
```

Comment:
is this actually a fontawesome text or something else? dont need to have the fa-icon in the classname imo

Reply:
I have updated the class name xD